src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -689,6 +698,15 @@ def setSiteWhitelist(self, siteWhitelist):
 
         return
 
+    def getSiteBlacklist(self):
+        """
+        Get the site black list from our data
+        :return: site black list
+        """
+        if getattr(self.data, 'SiteBlackList', None):
```

Comment:
By the way, I think this code can be resumed to:
```
return getattr(self.data, "SiteBlackList", [])
```
similarly for the whitelist.